lease/lessor.go

```diff
@@ -160,9 +164,10 @@ func newLessor(b backend.Backend, minLeaseTTL int64) *lessor {
 		b:           b,
 		minLeaseTTL: minLeaseTTL,
 		// expiredC is a small buffered chan to avoid unnecessary blocking.
-		expiredC: make(chan []*Lease, 16),
-		stopC:    make(chan struct{}),
-		doneC:    make(chan struct{}),
+		expiredC:  make(chan []*Lease, 16),
+		stopC:     make(chan struct{}),
+		doneC:     make(chan struct{}),
+		promoting: true,
```

Comment:
when there is a new leader, the new leader will call promote to refresh all its leases, right? for each refresh caused by election, we need the randomization.